docs/xroad_security_server_toolkit_user_guide.md

```diff
@@ -880,6 +882,21 @@ Access rights for a single endpoint can be add with:
 xrdsst endpoint add-access
 ```
 
+#### 4.2.8 Member management
+
+Members are managed with ``xrdsst member`` subcommands.
+
+There are no configuration parameters involved, command line arguments are used instead
+
+##### 4.2.8.1 Member find
+
+* Access rights: XROAD_SYSTEM_ADMINISTRATOR and XROAD_SERVICE_ADMINISTRATOR
```

Comment:
Please review the access rights, i think the XROAD_SERVICE_ADMINISTRATOR role is not required